recipes/intxx/bld.bat

```diff
@@ -0,0 +1,31 @@
+
+cmake %CMAKE_ARGS% ^
+  -G "Ninja" ^
+  -S %SRC_DIR% ^
+  -B build ^
+  -D CMAKE_BUILD_TYPE=Release ^
+  -D CMAKE_INSTALL_PREFIX="%PREFIX%" ^
+  -D CMAKE_INSTALL_LIBDIR="Library\lib" ^
+  -D CMAKE_INSTALL_INCLUDEDIR="Library\include" ^
+  -D CMAKE_INSTALL_BINDIR="Library\bin" ^
+  -D CMAKE_INSTALL_DATADIR="Library\share" ^
+  -D CMAKE_CXX_COMPILER=clang-cl ^
+  -D CMAKE_CXX_FLAGS="%CXXFLAGS%" ^
```

Comment:
```suggestion
```
Please add a conda_build_config.yaml where you set c(xx)_compiler to clang-cl for win

Reply:
Ok, I got the usual `lld-link: error: could not open '/link': no such file or directory` when setting clang-cl in this way. I'm getting this everywhere, so posted an inquiry at https://github.com/conda-forge/flang-activation-feedstock/issues/22 . For now, it's fixed by filtering `%LDFLAGS%`.